slices/gcc-12-base.yaml

```diff
@@ -0,0 +1,6 @@
+package: gcc-12-base
+
+slices:
+  copyright:
+    # we do not need to list the copyright file
+    # since chisel will add it anyway.
```

Comment:
Can you list the copyright path explicitly? It is always better to do so and eventually we plan on Chisel not installing the copyright by default.

Reply:
I don't understand what you mean by listing the copyright path explicitly?

Reply:
it's the same suggestion as in here -> https://github.com/canonical/chisel-releases/pull/168#discussion_r1603116388